{{cookiecutter.github_repository_name}}/docs/api/users.md

```diff
@@ -66,11 +44,6 @@ Content-Type application/json
 
 Parameters:
 
-Name | Type | Description
----|---|---
-id | integer | The id associated with the user object.
```

Comment:
Yeah so `id` is part of the path, not a parameter so removed the parameters section.

It should be clear from the context that `id` is a uuid, right?